features/hydra/error.feature

```diff
@@ -103,23 +103,39 @@ Feature: Error handling
       And the JSON node "hydra:description" should be equal to "Update is not allowed for this operation."
       And the JSON node "trace" should exist
 
-    Scenario: Get an error during update of an existing relation with a non-allowed update operation
+    @dropSchema
```

Comment:
You can remove this one, but be sure to add `@dropSchema` on the last scenario to cleanup.

Reply:
@dunglas I missed the fact that `@dropSchema` is `AfterScenario`.

I took the liberty of adding a new `@recreateSchema` that drop and create the schema, because I don't think relying on what did appear on the previous scenario is a good think. (mainly because someone may add another scenario between.

Reply:
Even if I agree that it's cleaner, rhe problem with recreating the schema often is that is slow down all our builds...
If someone add another scenario between, it's his build that will fail 😄. His responsibility.

Would you mind to just use create/drop?

Reply:
@dunglas I did that, and opened #1642 to discuss about the createSchema vs recreateSchema